wizeline/wize-q-front
# Feat - WIP-325 - Add who answered a question

#### What does this PR do?
- Displays the name of the person that answered a question if the name exists in the DB.

#### Where should the reviewer start?
- In src/components/LisQuestions.js check on line 215 renderAnswerRow.

#### How should this be manually tested?
1. Create an answer to a question and see in Answered Questions tabs if the answer has name.

#### Any background context you want to provide?
Depends on https://github.com/wizeline/wize-q-back/pull/50

#### What are the relevant tickets?
https://wizeline.atlassian.net/browse/WIP-325
